lib/extras/dec/decode.cc

```diff
@@ -55,7 +45,7 @@ Codec CodecFromPath(std::string path, size_t* JXL_RESTRICT bits_per_sample,
   std::string ext;
   BasenameAndExtension(path, &base, &ext);
   if (filename) *filename = base;
```

Comment:
like wise in this function, we don't need the filename output any more.

Reply:
also done